nc_test/run_t_nc.in

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# This script runs some PnetCDF I/O tests
+
+set -e
+@MPIEXEC@ -n 1 ./t_nc
```

Comment:
I see that we do need to call these tests with mpiexec -n 1 when pnetcdf is enabled. I think, however, it could all be done with one script. I will submit a PR.

Thanks for finding this.